guide/coaching.md

```diff
@@ -40,42 +44,59 @@ permalink: /guide/coaching/
 
 <p>There are a lot of good reasons to become a coach. First and foremost, you will be giving back to the community for a great cause: by empowering women to contribute to open source. It's a great way to share your knowledge. We also believe that coaching can be really rewarding for your own work, that it can improve the way you write code and help you grow as a developer, as well as make you think about concepts you take for granted. And above all, we promise you an overwhelming buzz looking into the happy faces of your students when they succeed.</p>
 
-<h3 id="what">What does coaching require?</h3>
+<h3 id="remote">Can I coach remotely?</h3>
+
+<p>The short answer is:</p>
+
+* Yes
+
+
+<p>We highly recommend students and coaches to spend their summer of code in one location, ideally sharing one working space. We believe that the ability to sit together in front of the screen can make a more beneficial and fruitful tandem. However, sometimes it’s not possible, and then remote coaching comes as a good solution.</p>
```

Comment:
change wording to:
We highly recommend that students and coaches spend their summer of code in one location, ideally sharing one workspace. We believe that sitting together in front of a screen with your team to help them solve problems or explain concepts is often more beneficial (and easier), both to new and seasoned learners. Sometimes, this just isn't possible—and that's when remote coaching becomes a good, alternative solution.